src/ex3/services.js

```diff
@@ -0,0 +1,130 @@
+import {
```

Comment:
please rename the file. services is not meaningful name.
This is handling the logic of adding/removing item so ItemManager should be more intuitive